tools/vera++/scripts/rules/jerry_dereference_operator_awlays_on_right.tcl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/tclsh
```

Comment:
As far as I know this is the correct style that should be followed. Do you know any exceptions?